pypy2/python.pypy.nuspec

```diff
@@ -3,7 +3,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>python.pypy</id>
-    <version>7.3.1</version>
+    <version>7.3.1.20200825</version>
```

Comment:
This version is just aesthetic / doesn't correspond to a "real" PyPy version?

Reply:
It's the same scheme as cpython versioning. Eg https://chocolatey.org/packages/python/3.8.5.20200721 - last part is a tag date.